secure-releases.md

```diff
@@ -0,0 +1,60 @@
+
+## CVE Management
+
+### Security Policy
+
+Your software project will, likely inevitably, have a security vulnerability. When (not if) that happens, you want to make sure that whoever discovers it has a way to report it to you privately - this is called "responsible disclosure" or "coordinated disclosure". So, the first important step is to *have a security policy*.
+
+On Github, this is always located at `$REPO_URL/security/policy` ([example](https://github.com/nvm-sh/nvm/security/policy)). This page can be populated from a number of file locations: `SECURITY.md` or `.github/SECURITY.md` in the repo itself, or, one of the same files in your user or organization’s `.github` repository. If you're not on Github, then look for a similar platform-specific convention, and if you find none, a top-level `SECURITY.md` is safe.
+
+In this policy, you'll want to mention a private reporting channel. This could be an email address, a service like HackerOne, etc.
+
+#### Programmatic Private Vulnerability Reporting
+
+Github and Gitlab support a Private Vulnerability Reporting (PVR) feature and API. This allows reporters to use the platform itself, programmatically if needed. It is strongly recommended to enable this feature, if it's not already on by default.
+
+### Audit Dependencies
+
+You'll want to ensure all of your dependencies - direct and transitive - are audited on every commit. This can be achieved locally and in CI with `npm audit`, if your project has a lockfile, or https://npmjs.com/aud for packages without a lockfile.
+
+In addition - not instead - you may want to use services like Dependabot, Renovate, [Tidelift](https://tidelift.com), [Socket.dev](https://socket.dev), etc to automatically detect and warn you about issues with your dependency graph.
+
+If your package does not have a build process - meaning, all of the artifacts you publish are committed to version control and hand-written - then you may only need to audit your runtime dependencies (`dependencies`, `peerDependencies`) and not `devDependendies`. However, if you do have a build process, whichever dev deps that generate code need to be audited as well.
+
+### Handling Reports
+
+First, be highly responsive. Ideally, you can respond to a reporter within 24 hours or 1 business day. Your security policy should explicitly describe your expected response time - especially if it's longer than this.
```

Comment:
```suggestion
First, be highly responsive. Respond to a report as soon as possible; ideally, within 24 hours or one business day. Your security policy should explicitly describe your expected response time - especially if it's longer than this.
```

Reply:
why 24 but "one"? generally i keep numbers as digits or words consistent throughout the document.

Reply:
most writing style guides spell out number one through nine or ten. some even continue spelling them out to 100. but generally they agree that single digits get spelled out

Reply:
not sure if we need to follow any extant writing style guides, but i'll make the change